octopusdeploy/schema_machine_update_policy.go

```diff
@@ -71,5 +76,14 @@ func getMachineUpdatePolicySchema() map[string]*schema.Schema {
 				"Update",
 			}, false)),
 		},
+		"kubernetes_agent_update_behavior": {
```

Comment:
It would be good to add a description so that the docs are a bit nicer but since the rest of the machine policy schema doesn't have any descriptions it probably doesn't make sense to add one unless you add descriptions to the rest of the attributes. Ill leave it up to you to decide if you want to do that or not

Reply:
I'll take a look and see how much work it is